content/participate/report-issue/redirect.html.haml

```diff
@@ -80,13 +80,20 @@ uneditable: true
                 Report a security issue privately.
                 %a{:href => '/security/reporting/' }
                     Learn more...
-            %a.app-button.app-button--security.bl{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10180&issuetype=10103&components=17329&summary=[INSERT_PLUGIN_ID_HERE]%20Insert%20Summary%20Here' }
+            %a.app-button.app-button--security.bl#link-security{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10180&issuetype=10103&components=17329&summary=[INSERT_PLUGIN_ID_HERE]%20Insert%20Summary%20Here' }
                 Continue
 
 :javascript
     // TODO Also pass the plugin name and show it on the UI, but for now this is simpler and probably safer.
-    var hash = window.location.hash.substring(1);
-    if (hash && /^\d+$/.test(hash)) {
+    let hash = window.location.hash.substring(1);
+    if (hash && /^\d+\/[a-zA-Z0-9_-]+$/.test(hash)) {
+        let parts = hash.split('/');
+        let componentId = parts[0];
+        let pluginId = parts[1];
```

Comment:
```suggestion
        const [componentId, pluginId] = hash.split('/');
```

Reply:
also you can first do the split, which may leave pluginId undefined if there is no `/`, then check that `componentId` consists of digits and not duplicate the link updates